docs/reference/lambda-support.md

```diff
@@ -0,0 +1,261 @@
+5. Click the *Add* button
+::::::
+
+::::::{tab-item} AWS CLI
+To add the Layer ARNs of the {{apm-lambda-ext}} and the APM agent through the AWS command line interface execute the following command:
+
+```bash
+aws lambda update-function-configuration --function-name yourLambdaFunctionName \
+--layers arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1 \ <1>
+arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-python-{{apm-python-v}}:1 <2>
+```
+1. Replace `{AWS_REGION}` with the AWS region of your Lambda function and `{ARCHITECTURE}` with its architecture.
+2. Replace `{AWS_REGION}` with the AWS region of your Lambda function.
+::::::
+
+::::::{tab-item} SAM
+In your SAM `template.yml` file add the Layer ARNs of the {{apm-lambda-ext}} and the APM agent as follows:
+
+```yaml
+...
+Resources:
+  yourLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      ...
+      Layers:
+          - arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1 <1>
+          - arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-python-{{apm-python-v}}:1 <2>
+...
+```
+1. Replace `{AWS_REGION}` with the AWS region of your Lambda function and `{ARCHITECTURE}` with its architecture.
+2. Replace `{AWS_REGION}` with the AWS region of your Lambda function.
+::::::
+
+::::::{tab-item} Serverless
+In your `serverless.yml` file add the Layer ARNs of the {{apm-lambda-ext}} and the APM agent to your function as follows:
+
+```yaml
+...
+functions:
+  yourLambdaFunction:
+    handler: ...
+    layers:
+      - arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1 <1>
+      - arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-python-{{apm-python-v}}:1 <2>
+...
+```
+1. Replace `{AWS_REGION}` with the AWS region of your Lambda function and `{ARCHITECTURE}` with its architecture.
+2. Replace `{AWS_REGION}` with the AWS region of your Lambda function.
+::::::
+
+::::::{tab-item} Terraform
+To add the{{apm-lambda-ext}} and the APM agent to your function add the ARNs to the `layers` property in your Terraform file:
+
+```yaml
+...
+resource "aws_lambda_function" "your_lambda_function" {
+  ...
+  layers = ["arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-extension-{{apm-lambda-ext-v}}-{ARCHITECTURE}:1", "arn:aws:lambda:{AWS_REGION}:267093732750:layer:elastic-apm-python-{{apm-python-v}}:1"] <1>
+}
+...
+```
+1. Replace `{AWS_REGION}` with the AWS region of your Lambda function and `{ARCHITECTURE}` with its architecture.
+::::::
+
+::::::{tab-item} Container Image
+To add the {{apm-lambda-ext}} and the APM agent to your container-based function extend the Dockerfile of your function image as follows:
+
+```Dockerfile
+FROM docker.elastic.co/observability/apm-lambda-extension-IMAGE_ARCH:latest AS lambda-extension
```

Comment:
This IMAGE_ARCH is missing a note that it should be substituted

Reply:
I added a code annotation (https://github.com/elastic/apm-agent-python/pull/2221/commits/ff98aa6a323e69e9e4d7f5a069d51d2a095983cf). Feel free to suggest alternative language for the note.